src/content/contributing/learning-quizzes/index.md

```diff
@@ -0,0 +1,70 @@
+
+Questions are structured as follows. The question prompt, 1 correct answer with an explanation of why it is correct, 3 incorrect answers with an explanation of why they are incorrect.
+
+Some examples of current quizzes can be found here:
+
+- [Layer 2](/layer-2)
+- [NFT](/nft/)
+- [What is Ethereum?](/what-is-ethereum/)
+- [What is ETH?](/eth/)
+
+## Adding a learn quiz
+
+If there is a page that hasn't had a learn quiz created for it, please [open an issue](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=&template=suggest_quiz.yaml) for it.
+
+Please provide the following information:
+
+- The page you want to add a quiz on
+- 5 questions with the following information:
+  - The section of the page the question is based on
+  - Question prompt
+  - 1 correct answer with an explanation for why it is correct
+  - 3 incorrect answers, each with an explanation for why they are incorrect
+
+## Adding a quiz question
+
+If there is a question you want to add to the question bank for a quiz, please [open an issue](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=&template=suggest_quiz.yaml) and provide the following information:
+
+- The page you want to add a quiz question on
+- For each question provide the following information:
+  - The section of the page the question is based on
+  - Question prompt
+  - 1 correct answer with an explanation for why it is correct
+  - 3 incorrect answers, each with an explanation for why they are incorrect
+
+## Updating a quiz question
+
+If there is a question you want to update in a question bank for a quiz, please [open an issue](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=&template=suggest_quiz.yaml) and provide the following information:
+
+- The page you want to update a quiz question on
+- For each question being updated, provide the following information:
+  - The section of the page the question is based on
+  - Question prompt of the question you want to update
+  - Updated question prompt
+  - 1 correct answer with an explanation for why it is correct
+  - 3 incorrect answers, each with an explanation for why they are incorrect
+
+## Removing a quiz question
+
+If the content no longer exists on the page for a question and it needs to be removed, please [open an issue](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=&template=suggest_quiz.yaml) to remove the question and provide the following information:
+
+- The page you want to delete a quiz question on
+- The question that you want to delete
+- Explanation if necessary for why the question should be removed
+
+## Contribute
+
+If you want to add, update, or delete quiz content for ethereum.org, create an issue on GitHub.
+
+<ButtonLink to="https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=&template=suggest_quiz.yaml">
+  Create an issue
+</ButtonLink>
```

Comment:
1. Isnt this section a bit redundant since the content of each above paragraphs specifically funnels the different scenarios? or am i missing any other purpose of this specific section?
2. the buttons could be used in each section instead of hyperlink to GH (easier to click on, more focus)

Reply:
No strong feeling about this section needing to be here, was just trying to follow previous patterns. Ill get rid of it for now as there are many links throughout each section.